src/bitbucket/BitbucketClient.ts

```diff
@@ -15,11 +15,11 @@ export class BitbucketClient {
 
   constructor(private config: Config) {}
 
-  async isAllowedToLand(pullRequestId: number) {
+  async isAllowedToLand(pullRequestId: number, permissionLevel: IPermissionMode) {
     const pullRequest: BB.PullRequest = await this.bitbucket.getPullRequest(pullRequestId);
     const buildStatuses = await this.bitbucket.getPullRequestBuildStatuses(pullRequestId);
     const author = pullRequest.author;
-    let approvals = getRealApprovals(
+    const approvals = getRealApprovals(
```

Comment:
Are we sure we are not calling approvals elsewhere?

Reply:
Definitely not.